aspnetcore/host-and-deploy/visual-studio-publish-profiles.md

```diff
@@ -396,25 +396,34 @@ Done Building Project "C:\Webs\Web1\Web1.csproj" (default targets).
 
 ## Include files
 
-The following markup:
+### Simple file inclusion 
 
-* Includes an *images* folder outside the project directory to the *wwwroot/images* folder of the publish site.
-* Can be added to the *.csproj* file or the publish profile. If it's added to the *.csproj* file, it's included in each publish profile in the project.
+The following example's `<ItemGroup>` element demonstrates copying a folder located outside of the project directory to a folder of the publish site. Any files added to the following markup's `<ItemGroup>` are included by default.
 
 ```xml
 <ItemGroup>
   <_CustomFiles Include="$(MSBuildProjectDirectory)/../images/**/*" />
-  <DotnetPublishFiles Include="@(_CustomFiles)">
+  <DotNetPublishFiles Include="@(_CustomFiles)">
     <DestinationRelativePath>wwwroot/images/%(RecursiveDir)%(Filename)%(Extension)</DestinationRelativePath>
-  </DotnetPublishFiles>
+  </DotNetPublishFiles>
 </ItemGroup>
 ```
 
-The following highlighted markup shows how to:
+The preceding markup:
+
+* Can be added to the *.csproj* file or the publish profile. If it's added to the *.csproj* file, it's included in each publish profile in the project.
+* Declares a `<_CustomFiles>` item to store files matching the `Include` attribute's globbing pattern. The *images* folder referenced in the pattern is located outside of the project directory. A [reserved property](/visualstudio/msbuild/msbuild-reserved-and-well-known-properties), named `$(MSBuildProjectDirectory)`, resolves to the project file's absolute path.
+* Uses the `DotNetPublishFiles` item, which is provided by a publish targets file in the Web SDK. The item accepts a list of files. The item's definition contains:
+  * An `<DestinationRelativePath>` element whose inner text is empty. The inner text is overridden in the markup and uses [well-known item metadata](/visualstudio/msbuild/msbuild-well-known-item-metadata) such as `%(RecursiveDir)`. The inner text represents the *wwwroot/images* folder of the publish site.
```

Comment:
```suggestion
 * A `<DestinationRelativePath>` element whose inner text is empty. The inner text is overridden in the markup and uses [well-known item metadata](/visualstudio/msbuild/msbuild-well-known-item-metadata), such as `%(RecursiveDir)`. The inner text represents the *wwwroot/images* folder of the published site.
```

Reply:
In general, I would prefer not mentioning the definition explicitly, instead referring to the "default behavior" of the item. Here, I don't think it's interesting that the default is empty, but it _is_ interesting to have an example of specifying the `DestinationRelativePath`.